python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_p2p_links.schema.yml

```diff
@@ -132,6 +132,23 @@ $defs:
               type: bool
               default: false
               description: Enable PTP.
+            role:
```

Comment:
please change to `roles` since it is a list. Sorry for not catching earlier.

Reply:
No problem. Changed to roles.